assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/components/TicketPriceCalculatorModal.tsx

```diff
@@ -1,46 +1,23 @@
-import React, { useCallback } from 'react';
+import React from 'react';
 import { __, sprintf } from '@wordpress/i18n';
-import { ButtonProps } from 'antd/lib/button';
-import { SaveOutlined } from '@ant-design/icons';
 import { Modal } from 'antd';
 
+import useResetButtonProps from '../buttons/useResetButtonProps';
+import useSubmitButtonProps from '../buttons/useSubmitButtonProps';
 import TicketPriceCalculator from './TicketPriceCalculator';
-import { useTPCContext } from '../context';
 import { useOnSubmitPrices } from '../hooks';
+import { useTPCContext } from '../context';
 
 import './styles.scss';
 
 const TicketPriceCalculatorModal: React.FC = () => {
 	const {
 		onCloseModal,
-		dataState: { ticket, reset },
+		dataState: { reset, ticket },
 	} = useTPCContext();
-
 	const submitPrices = useOnSubmitPrices();
```

Comment:
might be better located in `useSubmitButtonProps()`

Reply:
good catch, thx :bowing_man: